docs/docs-beta/docs/guides/deploy/deployment-options/aws.md

```diff
@@ -0,0 +1,252 @@
+    container_name: "my_container_name"
+```
+
+### Customizing CPU, memory, and ephemeral storage in ECS
+
+You can set the `run_launcher.config.run_resources` field to customize the default resources for Dagster runs. For example:
+
+```yaml
+run_launcher:
+  module: "dagster_aws.ecs"
+  class: "EcsRunLauncher"
+  config:
+    run_resources:
+      cpu: "256"
+      memory: "512" # In MiB
+      ephemeral_storage: 128 # In GiB
+```
+
+:::note
+
+Fargate tasks only support [certain combinations of CPU and memory](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task-cpu-memory-error.html) and values of ephemeral storage between 21 and 200 GiB (the default is set to 20 GiB).
+
+:::
+
+You can also use job tags to customize the CPU, memory, or ephemeral storage of every run for a particular job:
+
+```py
+from dagster import job, op
+
+@op()
+def my_op(context):
+  context.log.info('running')
+
+@job(
+  tags = {
+    "ecs/cpu": "256",
+    "ecs/memory": "512",
+    "ecs/ephemeral_storage": "40",
+  }
+)
+def my_job():
+  my_op()
+```
+
+If these tags are set, they will override any defaults set on the run launcher.
+
+### Customizing the launched run's task
+
+The <PyObject section="libraries" module="dagster_aws" object="ecs.EcsRunLauncher" /> creates a new task for each run, using the current ECS task to determine network configuration. For example, the launched run will use the same ECS cluster, subnets, security groups, and launch type (e.g. Fargate or EC2).
+
+To adjust the configuration of the launched run's task, set the `run_launcher.config.run_task_kwargs` field to a dictionary with additional key-value pairs that should be passed into the `run_task` boto3 API call. For example, to launch new runs in EC2 from a task running in Fargate, you could apply this configuration:
+
+```yaml
+run_launcher:
+  module: "dagster_aws.ecs"
+  class: "EcsRunLauncher"
+  config:
+    run_task_kwargs:
+      launchType: "EC2"
+```
+
+or to set the capacity provider strategy to run in Fargate Spot instances:
+
+```yaml
+run_launcher:
+  module: "dagster_aws.ecs"
+  class: "EcsRunLauncher"
+  config:
+    run_task_kwargs:
+      capacityProviderStrategy:
+        - capacityProvider: "FARGATE_SPOT"
+```
+
+You can also use the `ecs/run_task_kwargs` tag to customize the ECS task of every run for a particular job:
+
+```py
+from dagster import job, op
+
+@op()
+def my_op(context):
+  context.log.info('running')
+
+@job(
+  tags = {
+    "ecs/run_task_kwargs": {
+      "capacityProviderStrategy": [
+        {
+          "capacityProvider": "FARGATE_SPOT",
+        },
+      ],
+    },
+  }
+)
+def my_job():
+  my_op()
+```
+
+Refer to the [boto3 docs](https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/ecs.html#ECS.Client.run_task) for the full set of available arguments to `run_task`. Additionally, note that:
+
+- Keys are in `camelCase` as they correspond to arguments in boto's API
```

Comment:
Does `camelCase` need to be formatted as code?

Reply:
No, I'll remove that formatting.